lib/resources/gen/users.js

```diff
@@ -61,6 +61,28 @@ Users.prototype.findById = function(
   return this.dispatchGet(path, params, dispatchOptions);
 };
 
+/**
+ * Returns all of a user's favorites in the given workspace, of the given type.
+ * Results are given in order (The same order as Asana's sidebar).
+   * @param {String} user An identifier for the user. Can be one of an email address,
+   * the globally unique identifier for the user, or the keyword `me`
+   * to indicate the current user making the request.
+   * @param {Object} [params] Parameters for the request
+   * @param {String} params.workspace The workspace in which to get favorites.
+   * @param {String} params.resource_type The resource type of favorites to be returned.
+   * @param {Object} [dispatchOptions] Options, if any, to pass the dispatcher for the request
+   * @return {Promise} The requested resource
+ */
+Users.prototype.getUserFavorites = function(
```

Comment:
The fun of javascript... I'm having trouble coming up with a test that will work for this since there are not static types, and I'm not hitting any data. The tests can only really confirm which url a function is trying to hit and that parameters would get through.